package/run.sh

```diff
@@ -18,6 +19,13 @@ fi
 
 env "INSTALL_RKE2_ARTIFACT_PATH=${CATTLE_AGENT_EXECUTION_PWD}" installer.sh
 
+if [ ! -f ${RKE2_ENV_FILE} ]; then
+    install -m 600 /dev/null ${RKE2_ENV_FILE}
+fi
+
+env | grep '^RKE2_' | tee -a ${RKE2_ENV_FILE} >/dev/null
```

Comment:
Is `tee` necessary here? We don't use it anywhere else i.e. RKE2 install or this script and we don't necessarily know it will exist on our destination system.